### PR TITLE
Clean Arc build jsons

### LINKED ISSUE #7: The `@daostack/migration` package requires `@daostack/arc`
Because of the Arc dependency whoever installs the package will have to download megabytes of truffle JSON files.
I suggest splitting the package into two packages, one just containing the `migration.json` result file and the other containing all the migration utilities. A question is where the ganache `db` fits.

### PR DESCRIPTION
**Pull request by ben-kaufman**
Fix #7

### CODE REVIEW

File: pruneArcBuild.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  nodir: true
+});
+
+console.log(`Starts pruning Arc JSON files`);
+
+files.filter(file => {
+  const contract = require(`${file}`);
+  console.log(`Pruning ${contract.contractName}`);
+  var cleanVersion = '{\n "contractName": "';
+  cleanVersion += contract.contractName;
```
Comment (tsuberim) [inline pruneArcBuild.js]: Seems like a weird and error prone way to do it. I would:
```
const {contractName, abi, bytecode, deployedBytecode} = JSON.parse(fileContent)
fs.writeFileSync(pathToFile, JSON.stringify({contractName, abi, bytecode, deployedBytecode}, undefined, 2), 'utf-8')
```

File: package.json [hunk 2]
```diff
@@ -36,7 +37,9 @@
     "moment": "^2.22.2",
     "ora": "^3.0.0",
     "prompt-confirm": "^2.0.4",
+    "typedarray-to-buffer": "^3.1.5",
     "web3": "1.0.0-beta.35",
+    "yaeti": "^1.0.2",
```
Comment (tsuberim) [inline package.json]: why is this needed?

File: package.json [hunk 3]
```diff
@@ -36,7 +37,9 @@
     "moment": "^2.22.2",
     "ora": "^3.0.0",
     "prompt-confirm": "^2.0.4",
+    "typedarray-to-buffer": "^3.1.5",
```
Comment (tsuberim) [inline package.json]: why is this needed?

File: package.json [hunk 4]
```diff
@@ -15,7 +15,8 @@
     "migrate": "./migrate.js",
     "docker:build": "docker build . -t daostack/migration:$(cat package.json | jq -r '.version')",
     "docker:push": "docker push daostack/migration:$(cat package.json | jq -r '.version')",
-    "release": "./release.sh"
+    "release": "./release.sh",
+    "prune-arc-build": "node ./pruneArcBuild.js"
```
Comment (tsuberim) [inline package.json]: Maybe better to add it to `postinstall` script or similar, so it happens automatically whenever you `npm install` the repo.

File: pruneArcBuild.js [hunk 5]
```diff
@@ -0,0 +1,27 @@
+const glob = require("glob");
+var fs = require("fs");
+
+const files = glob.sync("./node_modules/@daostack/arc/build/contracts/*.json", {
+  nodir: true
+});
+
+console.log(`Starts pruning Arc JSON files`);
```
Comment (tsuberim) [inline pruneArcBuild.js]: Suggestion: could use `ora` to make the prints more pretty.

File: package.json (hunk 2 above)
Comment (ben-kaufman) [reply package.json]: It crashed and required to install that for some reason

File: package.json (hunk 3 above)
Comment (ben-kaufman) [reply package.json]: It crashed and required to install that for some reason

File: pruneArcBuild.js (hunk 1 above)
Comment (ben-kaufman) [reply pruneArcBuild.js]: Fixed

File: pruneArcBuild.js (hunk 5 above)
Comment (ben-kaufman) [reply pruneArcBuild.js]: Will add

File: package.json (hunk 4 above)
Comment (ben-kaufman) [reply package.json]: Fixed

File: pruneArcBuild.js (hunk 5 above)
Comment (ben-kaufman) [reply pruneArcBuild.js]: Fixed

File: package.json (hunk 2 above)
Comment (tsuberim) [reply package.json]: What crashed and why?

Comment (ben-kaufman) [reply package.json]: internal/modules/cjs/loader.js:589
    throw err;
    ^

Error: Cannot find module 'typedarray-to-buffer'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:587:15)
    at Function.Module._load (internal/modules/cjs/loader.js:513:25)
    at Module.require (internal/modules/cjs/loader.js:643:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at Object.<anonymous> (/Users/benkaufman/Documents/Work/DAOstack/Migration/node_modules/web3-providers-ws/node_modules/websocket/lib/W3CWebSocket.js:18:16)
    at Module._compile (internal/modules/cjs/loader.js:707:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:718:10)
    at Module.load (internal/modules/cjs/loader.js:605:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:544:12)
    at Function.Module._load (internal/modules/cjs/loader.js:536:3)

Comment (ben-kaufman) [reply package.json]: When running `npm run migrate`

Comment (tsuberim) [reply package.json]: and it didn't happen before this PR?

Comment (ben-kaufman) [reply package.json]: It happened when I started working on the PR

Comment (tsuberim) [reply package.json]: So something in this PR requires those package?

Comment (ben-kaufman) [reply package.json]: After running npm install again it worked, strange.
Removed that.
